README_GPDB_ru.md

```diff
@@ -0,0 +1,222 @@
+Данная инструкция описывает, как скомпилировать pgBackRest, создать с его помощью резервную копию кластера Greenplum и восстановиться из нее.
+
+# 1. Поддерживаемые версии Greenplum
+
+На данный момент поддерживается только [Greenplum 6](https://docs.vmware.com/en/VMware-Greenplum/6/greenplum-database/landing-index.html).
+
+# 2. Компиляция pgBackRest
+
+- Установить зависимости
+
+Приведенные ниже команды требуется выполнить от имени суперпользователя.
+
+для CentOS 7:
+```
+yum install git gcc openssl-devel libxml2-devel bzip2-devel libzstd-devel lz4-devel libyaml-devel
+```
+
+для ALT Linux p10
+```
+apt-get update
+apt-get install git gcc openssl-devel libxml2-devel bzip2-devel libzstd-devel libyaml-devel zlib-devel 
+```
+
+- Добавить в переменную окружения PATH путь к pg_config Greenplum
```

Comment:
As discussed offline, it may be not enough, as it looks that LD_LIBRARY_PATH is also required